scripts/release/update-comment.js

```diff
@@ -1,27 +1,35 @@
-#!/usr/bin/env node
-
 const fs = require('fs');
-const glob = require('glob');
 const proc = require('child_process');
+const semver = require('semver');
+const run = (cmd, ...args) => proc.execFileSync(cmd, args, { encoding: 'utf8' }).trim();
 
-const gitStatus = proc.execFileSync('git', ['status', '--porcelain', '-uno', 'contracts/**/*.sol']);
-
+const gitStatus = run('git', 'status', '--porcelain', '-uno', 'contracts/**/*.sol');
 if (gitStatus.length > 0) {
   console.error('Contracts directory is not clean');
   process.exit(1);
 }
 
 const { version } = require('../../package.json');
 
-const files = glob.sync('contracts/!(mocks)/**/*.sol');
+// Get latest tag according to semver.
+const [ tag ] = run('git', 'tag')
+  .split(/\r?\n/)
+  .filter(v => semver.valid(v) && semver.lte(v, version))
+  .sort(semver.rcompare);
+
+// Ordering tag → HEAD is important here.
+// Is it right to use HEAD ?
+const files = run('git', 'diff', tag, 'HEAD', '--name-only', 'contracts/**/*.sol')
+  .split(/\r?\n/)
+  .filter(file => !file.match(/mock/i));
```

Comment:
Not all contracts in there are named `*Mock`. I think this is a better filter:

```suggestion
  .filter(file => !file.includes('/mocks/'));
```

Reply:
This does both, it looks for "mock" in an case-incensitive way. It will match ".../**mock**s/..." and also ".../Something**Mock**.sol.

I think this is more powerful.